migrations/Version202203151451263291_ltiTestReview.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\ltiTestReview\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use common_ext_Extension as Extension;
+use common_ext_ExtensionsManager as ExtensionsManager;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version202203151451263291_ltiTestReview extends AbstractMigration
+{
+    private const CONFIG_FILE = 'ReviewPanel';
+
+    public function getDescription(): string
+    {
+        return 'Enable item tooltip for ReviewPanel';
```

Comment:
```suggestion
        return 'Configure default item tooltip for ReviewPanel';
```
"Enable" looks a bit confusing, since the value you are setting below is `false`.

Reply:
I changed to "Set", "Disable" is better but also not true because it wasn't enabled before